vendor/modules.txt

```diff
@@ -301,7 +301,7 @@ github.com/v3io/frames
 github.com/v3io/frames/pb
 # github.com/v3io/v3io-go-http v0.0.0-20190221115935-53e2b487c9a2
 github.com/v3io/v3io-go-http
-# github.com/v3io/v3io-tsdb v0.9.0 => github.com/v3io/v3io-tsdb v0.9.0
+# github.com/v3io/v3io-tsdb v0.9.0
```

Comment:
why not v0.9.2?

Reply:
I can do it, but it's not related to this PR :)